fields.yml

```
@@ -647,11 +647,7 @@
       title: Host
       group: 2
       description: >
-        Host fields provide information related to a host. A host can be a
-        physical machine, a virtual machine, or a Docker container.
-    
-        Normally the host information is related to the machine on which the event
-        was generated/collected, but they can be used differently if needed.
+        A host is defined as a general computing instance that is not a device. ECS host.* fields should be populated with details about the host on which the event happened, or on which the measurement was taken. Hosts types include hardware, virtual machines, docker containers, and kubernetes nodes.
```

```suggestion
        A host is defined as a general computing instance that is not a device. ECS host.* fields should be populated with details about the host on which the event happened, or on which the measurement was taken. Hosts types include hardware, virtual machines, Docker containers, and Kubernetes nodes.
```

Capitalized Docker and Kubernetes.